spec/features/competition_management_spec.rb

```diff
@@ -193,7 +193,8 @@ def wca_registration_checkbox
       competition = FactoryBot.create(:competition, :with_delegate, id: "OldId2016", guests_entry_fee_lowest_denomination: 666)
       visit edit_competition_path(competition)
 
-      expect(page).not_to have_text("Display message for free guest entry")
+      expect(page).to have_text(competition.name)
```

Comment:
Where did this come from? Looks like a new line

Reply:
Makes sense that the cop requires a positive assertion to succeed before we're "allowed" to make an assertion about the lack of presence of a particular element